src/i18n/DE/grid-resources.ts

```diff
@@ -0,0 +1,132 @@
+const GridResourceStringsDE_: ExpandRequire<IGridResourceStrings> = {
+  igx_grid_groupByArea_message: 'Lege eine Spaltenüberschrift hier ab, um danach zu sortieren.',
+  igx_grid_emptyFilteredGrid_message: 'Keine Einträge gefunden.',
+  igx_grid_emptyGrid_message: 'Grid hat keine Daten.',
+  igx_grid_filter: 'Filter',
+  igx_grid_filter_row_close: 'Schließen',
+  igx_grid_filter_row_reset: 'Zurücksetzen',
+  igx_grid_filter_row_placeholder: 'Füge Filterwert hinzu',
+  igx_grid_filter_row_boolean_placeholder: 'Alle',
+  igx_grid_filter_row_date_placeholder: 'Wähle Datum',
+  igx_grid_filter_operator_and: 'Und',
+  igx_grid_filter_operator_or: 'Oder',
+  igx_grid_complex_filter: 'Komplexer Filter',
+  igx_grid_filter_contains: 'Enthält',
+  igx_grid_filter_doesNotContain: 'Enthält nicht',
+  igx_grid_filter_startsWith: 'Beginnt mit',
+  igx_grid_filter_endsWith: 'Endet mit',
+  igx_grid_filter_equals: 'Gleich',
+  igx_grid_filter_doesNotEqual: 'Nicht gleich',
+  igx_grid_filter_empty: 'Leer',
+  igx_grid_filter_notEmpty: 'Nicht leer',
+  igx_grid_filter_null: 'Null',
+  igx_grid_filter_notNull: 'Nicht Null',
+  igx_grid_filter_before: 'Vor',
+  igx_grid_filter_after: 'Nach',
+  igx_grid_filter_today: 'Heute',
+  igx_grid_filter_yesterday: 'Gestern',
+  igx_grid_filter_thisMonth: 'Diesen Monat',
+  igx_grid_filter_lastMonth: 'Letzten Monat',
+  igx_grid_filter_nextMonth: 'Nächsten Monat',
+  igx_grid_filter_thisYear: 'Dieses Jahr',
+  igx_grid_filter_lastYear: 'Letztes Jahr',
+  igx_grid_filter_nextYear: 'Nächstes Jahr',
+  igx_grid_filter_greaterThan: 'Größer als',
+  igx_grid_filter_lessThan: 'Kleiner als',
+  igx_grid_filter_greaterThanOrEqualTo: 'Größer als oder gleich',
+  igx_grid_filter_lessThanOrEqualTo: 'Kleiner als oder gleich',
+  igx_grid_filter_true: 'Wahr',
+  igx_grid_filter_false: 'Falsche',
+  igx_grid_filter_all: 'Alle',
+  igx_grid_filter_condition_placeholder: 'Wähle Filter',
+  igx_grid_summary_count: 'Anzahl',
+  igx_grid_summary_min: 'Min',
+  igx_grid_summary_max: 'Max',
+  igx_grid_summary_sum: 'Summe',
+  igx_grid_summary_average: 'Mittel',
+  igx_grid_summary_earliest: 'Früheste',
+  igx_grid_summary_latest: 'Späteste',
+  igx_grid_excel_filter_moving_left: 'Nach links',
+  igx_grid_excel_filter_moving_left_short: 'Links',
+  igx_grid_excel_filter_moving_right: 'Nach rechts',
+  igx_grid_excel_filter_moving_right_short: 'Rechts',
+  igx_grid_excel_filter_moving_header: 'Verschiebe',
+  igx_grid_excel_filter_sorting_asc: 'Aufsteigend',
+  igx_grid_excel_filter_sorting_asc_short: 'Auf',
+  igx_grid_excel_filter_sorting_desc: 'Absteigend',
+  igx_grid_excel_filter_sorting_desc_short: 'Ab',
+  igx_grid_excel_filter_sorting_header: 'Sortiere',
+  igx_grid_excel_filter_clear: 'Lösche Filter',
+  igx_grid_excel_custom_dialog_add: 'Füge Filter hinzu',
+  igx_grid_excel_custom_dialog_clear: 'Lösche Filter',
+  igx_grid_excel_custom_dialog_header: 'Auto-Filter für Spalte: ',
```

Comment:
```suggestion
  igx_grid_excel_custom_dialog_header: 'Benutzerdef. Auto-Filter für Spalte: ',
```

Reply:
Abbreviation seems odd to me, but I don't know whether the UI has sufficient space. If so, I'd use "Benutzerdefinierten Auto-Filter für Spalte: "

Reply:
I had it without abbreviation, but then saw it further down with, maybe both should contain the complete word?

Reply:
This is the only place where "Auto-Filter" is referred to. Everywhere else it is "Filter" (same in the English text btw). So I decided to split the difference as "Benutzerdefinierter Filter für Spalte: "